src/node/txreconciliation.cpp

```diff
@@ -142,9 +250,104 @@ class TxReconciliationTracker::Impl
         return (recon_state != m_states.end() &&
                 std::holds_alternative<TxReconciliationState>(recon_state->second));
     }
+
+    // Not const because of caching.
+    bool IsFanoutTarget(const Wtxid& wtxid, NodeId peer_id, bool we_initiate, double limit) EXCLUSIVE_LOCKS_REQUIRED(m_txreconciliation_mutex)
```

Comment:
This variable name can be called `destination` rather than `limit` to be consistent with `ShouldFanoutTo` and denotates more clearly it’s the sample space boundary.